pkgs/top-level/all-packages.nix

```diff
@@ -3504,6 +3504,7 @@ in
   gnupg22 = callPackage ../tools/security/gnupg/22.nix {
     pinentry = if stdenv.isDarwin then pinentry_mac else pinentry;
   };
+  gnupg-minimal = callPackage ../tools/security/gnupg/minimal.nix { };
```

Comment:
I don't think we should make this easily acessible in the top-level package set, better just pass the `gnupg.override {…}` into systemd'd `callPackage`, or do the override inside the systemd derivation itself.

Reply:
Which one do you prefer?

Reply:
personally, I'd go with defining an overridden gnupg in a let-expression inside the systemd derivation to not clutter `all-packages.nix` further…

However, people might want to pass another gnupg to build systemd, so it might make sense to do the override in `all-packages.nix`, and pass the overridden `gnupg =` down to `callPackage`.